MerlinAU.sh

```diff
@@ -1756,12 +1764,13 @@ _RunFirmwareUpdateNow_()
     Say "Required RAM: ${required_space_kb} KB - Available RAM: ${availableRAM_kb} KB"
     check_memory_and_prompt_reboot "$required_space_kb" "$availableRAM_kb"
 
+    trap "_DoCleanUp_ 0 "$keepZIPfile" ; exit 0" HUP INT QUIT TERM
```

Comment:
Thanks for catching the trap for the cleanup, smart to have it outside the versioning check.